Tasks/Rybina/Task1/Task1.py

```diff
@@ -0,0 +1,17 @@
+#variant1
+
+deposit = 20000
+period = 60
+rate = 15/12/100
+x = deposit*(rate + 1)**period
+print(x)
```

Comment:
the answer should be properly rounded to show 'kopeiki', -15 points for the logical mistake